swift/ios-shoppe-demo/Resources/AppDelegate.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  AppDelegate.swift
+//  ios-shoppe-demo
+//
+//  Created on 3/3/20.
+//  Copyright © 2020 FullStory All rights reserved.
+//
+
```

Comment:
I'm not used to seeing code in a "resources" directory. From [this](https://developer.apple.com/library/archive/documentation/CoreFoundation/Conceptual/CFBundles/BundleTypes/BundleTypes.html#//apple_ref/doc/uid/10000123i-CH101-SW1):

> Resources are data files that live outside your application’s executable file. Resources typically consist of things like images, icons, sounds, nib files, strings files, configuration files, and data files (among others)

Should `AppDelegate.swift` and `SceneDelegate.swift` live here?

Reply:
Maybe not. I moved them out.

Reply:
It looks like `AppDelegate.swift` and `SceneDelegate.swift (just noticed `Utilities.swift` as well) are still in the `Resources` directory:

![image](https://user-images.githubusercontent.com/45576380/76430789-19a1f280-6387-11ea-97d2-bac85d98699e.png)
![image](https://user-images.githubusercontent.com/45576380/76430815-2292c400-6387-11ea-8789-b321ffd2f579.png)
![image](https://user-images.githubusercontent.com/45576380/76430870-34746700-6387-11ea-89e5-2f98b54f2e82.png)

Maybe you forgot to commit the change?

Reply:
`Utilities.swift` is still in the `Resources` directory:

![image](https://user-images.githubusercontent.com/45576380/76435000-d64a8280-638c-11ea-97fd-69f6765486b2.png)

Reply:
It should be there its needed.